src/convert_case.rs

```diff
@@ -0,0 +1,101 @@
+use crate::kebab::to_kebab;
+use crate::lower_camel::to_lower_camel_case;
+use crate::shouty_kebab::to_shouty_kebab_case;
+use crate::shouty_snake::to_shouty_snake_case;
+use crate::snake::to_snake_case;
+use crate::title::to_title_case;
+use crate::upper_camel::to_upper_camel_case;
+
+/// This trait defines a wrapper function for other case-conversion functions
+/// in that this can tweak the behaviour of those functions based on customization options
+///
+///
+/// ## Example:
+/// ```rust
+///
```

Comment:
Swap these two lines (there should not be an empty line at the start of the code block).